src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/EditInvitation/EditInvitationCommandHandlerTests.cs

```diff
@@ -681,6 +668,10 @@ public async Task HandlingUpdateIpoCommand_ShouldSetAndReturnRowVersion()
             // Since UnitOfWorkMock is a Mock this will not happen here, so we assert that RowVersion is set from command
             Assert.AreEqual(_rowVersion, result.Data);
             Assert.AreEqual(_rowVersion, _dpInvitation.RowVersion.ConvertToString());
+            foreach (var participant in _dpInvitation.Participants)
```

Comment:
what does this do?

Reply:
Good catch! Just debug code I inserted while digging for reason on a red test. Removed now